app/services/github_checks_verifier.rb

```diff
@@ -47,6 +48,8 @@ def log_checks(checks, msg)
 
   def apply_filters(checks)
     checks.reject! { |check| check.name == workflow_name }
+    checks.reject! { |check| ignore_checks.include?(check.name) }
```

Comment:
In order to avoid iterating the array twice (by calling `reject`), I'd suggest joining both of them:
`checks.reject! { |check| [ignore_checks, workflow_name].flatten.include?(check.name) }`

Reply:
Thanks Done !